packages/console/src/pages/ApplicationDetails/components/DeleteForm/index.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react';
+import { toast } from 'react-hot-toast';
+import { Trans, useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router-dom';
+
+import Button from '@/components/Button';
+import ModalLayout from '@/components/ModalLayout';
+import TextInput from '@/components/TextInput';
+import useApi from '@/hooks/use-api';
+
+import * as styles from './index.module.scss';
+
+type Props = {
+  id: string;
+  name: string;
+  onClose: () => void;
+};
+
+const DeleteForm = ({ id, name, onClose }: Props) => {
+  const api = useApi();
+  const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
+
+  const navigate = useNavigate();
+
+  const [inputName, setInputName] = useState('');
+  const [loading, setLoading] = useState(false);
+
+  const inputMismatched = inputName !== name;
+
+  const handleDelete = async () => {
+    setLoading(true);
+
+    try {
+      await api.delete(`/api/applications/${id}`);
+      onClose();
+      navigate(`/applications`);
+      toast.success(t('application_details.application_deleted', { name }));
+    } finally {
+      setLoading(false);
+    }
+  };
+
+  return (
+    <ModalLayout
+      title="application_details.reminder"
+      content={() => (
+        <div className={styles.content}>
+          <div className={styles.description}>
+            <Trans
+              t={t}
+              i18nKey="application_details.delete_description"
+              values={{ name }}
+              components={{ span: <span className={styles.hightlight} /> }}
+            />
```

Comment:
What's the difference between this and the following:
```typescript
<span className={styles.highlight}>
  {t('application_details.delete_description', { name })}
</span>
```
This looks even cleaner to me

Reply:
We have styles in the i18n-text.
<img width="667" alt="image" src="https://user-images.githubusercontent.com/10806653/159212545-2653c161-c95a-4e65-a6ee-3f844e88f340.png">

Reply:
https://react.i18next.com/latest/trans-component

Reply:
ok, got it